autogen/agentchat/contrib/critic_agent.py

```diff
@@ -0,0 +1,55 @@
+from typing import Callable, Dict, Literal, Optional, Union
```

Comment:
in my opinion: critic_agent.py, quantifier_agent.py, subcritic_agent.py, should be in some agenteval subdirectory, specially if they remain under the /contrib directory. That also allows you to create a README.md file that can give some more background on the role of this agents.

Reply:
I can do that. @ekzhu any objections to adding folder/readme for organization?